docs/source/cpp/compute.rst

```diff
@@ -201,6 +201,11 @@ an ``Invalid`` :class:`Status` when overflow is detected.
 +--------------------------+------------+--------------------+---------------------+
 | subtract_checked         | Binary     | Numeric            | Numeric             |
 +--------------------------+------------+--------------------+---------------------+
+| divide                   | Binary     | Numeric            | Numeric             |
```

Comment:
Noticed that this was missing on the c++ page

Reply:
Ah, thank you! However, can you leave the table alphabetically-sorted?

Reply:
Actually my bad, you already had it there. Reverted